circle.yml

```diff
@@ -55,7 +59,21 @@ jobs:
           no_output_timeout: 1200
           shell: /bin/bash -euo pipefail
           command: |
-            gcloud firebase test android run --type robo --app app/build/outputs/apk/debug/app-debug.apk --device-ids shamu --os-version-ids 22 --locales en --orientations portrait --timeout 5m
+            gcloud firebase test android run --type robo \
+              --app app/build/outputs/apk/debug/app-debug.apk \
+              --device-ids shamu \
+              --os-version-ids 22 \
+              --locales en \
+              --orientations portrait \
+              --timeout 5m
+            gcloud firebase test android run --type instrumentation \
+              --app app/build/outputs/apk/debug/app-debug.apk \
+              --test app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk \
+              --device model=hammerhead,version=21,locale=en,orientation=portrait  \
```

Comment:
What about setting the locale to German instead of English? We're already testing `en` 👇